pkgs/build-support/checkpoint-build.nix

```diff
@@ -53,7 +53,12 @@ rec {
       mkdir -p $out/outputs
       cp -r ./* $out/outputs/
       runHook postCheckpointInstall
+      unset postPhases
```

Comment:
Should we also unset `preInstallPhases`, `preFixupPhases` and `preDistPhases`?

Reply:
Sounds good. `preInstallPhases` has no effect, because the Phase is already over, but the other ones can be unset